src/fromager/commands/build.py

```diff
@@ -127,13 +127,24 @@ def build_sequence(
                 wheel_filename = _build(
                     wkctx, resolved_version, req, source_download_url
                 )
+            packages_built.append(entry)
```

Comment:
If we're using the logger, and writing to a separate file, we could just log here instead of waiting until the end. That way even if the build fails, we'll have a nice summary of what we were able to do.

Reply:
I wonder how it will work with #429? Should we try to combine this with #429?